addons/mass_mailing/wizard/mail_compose_message.py

```diff
@@ -12,6 +12,12 @@ class MailComposeMessage(models.TransientModel):
     campaign_id = fields.Many2one('utm.campaign', string='Mass Mailing Campaign')
     mass_mailing_name = fields.Char(string='Mass Mailing Name')
     mailing_list_ids = fields.Many2many('mailing.list', string='Mailing List')
+    model_is_thread = fields.Boolean(compute='_compute_model_is_thread')
+
+    @api.depends('model')
+    def _compute_model_is_thread(self):
+        for composer in self:
+            composer.model_is_thread = bool(composer.env['ir.model'].sudo().search([('model', '=', composer.model), ('is_mail_thread', '=', True)], limit=1))
```

Comment:
can you move this search outside of the loop ? (better to never have a search in a loop)